core/src/Cuda/Kokkos_Cuda_View.hpp

```diff
@@ -139,7 +139,7 @@ struct CudaLDGFetch {
 
   template <typename iType>
   KOKKOS_INLINE_FUNCTION ValueType operator[](const iType& i) const {
-#ifdef __CUDA_ARCH__
+#if defined(__CUDA_ARCH__) && !defined(KOKKOS_ARCH_KEPLER30)
```

Comment:
```suggestion
#if defined(__CUDA_ARCH__) && (__CUDA_ARCH__ >= 350)
```

Reply:
320

Reply:
The [documentation](https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#ldg-function) says "only supported by devices of compute capability 3.5 and higher"